src/DIRAC/ConfigurationSystem/Client/LocalConfiguration.py

```diff
@@ -310,6 +392,38 @@ def __parseCommandLine(self):
           To modify the local configuration use '--cfg <configfile>' instead.""")
         if os.environ.get("DIRAC_DEPRECATED_FAIL", None):
           raise NotImplementedError("ERROR: using deprecated config file passing option.")
+
+    # Check arguments
+    if len(self.commandArgList) < len([t for t in self.commandArgumentList if t[2]]):
+      gLogger.fatal('Error when parsing command line arguments: not all mandatory arguments are defined.')
+      self.showHelp(exitCode=1)
+
+    groupArgs = []
+    step = 0
+    for i in range(len(self.commandArgumentList)):
+      argMarking, description, mandatory, values, default = self.commandArgumentList[i]
+
+      # Check whether the required arguments are given in the command line
+      if len(self.commandArgList) <= (i + step):
+        groupArgs.append(default)
+        continue
+
+      # Replace the default value with a defined one
+      if re.match(r"\w+ \[\w+\]", argMarking):  # When we consider multiple value argument
+        # The index of the last such argument
+        step = len(self.commandArgList) - len(self.commandArgumentList)
+        groupArgs.append(self.commandArgList[i:i + 1 + step])
+      else:
+        groupArgs.append(self.commandArgList[i + step])
+
+      # Check accepted values
+      for cArg in self.commandArgList[i:i + 1 + step]:
+        if values and cArg not in values:
+          gLogger.fatal('Error when parsing command line arguments: \
+                         "%s" does not match the allowed values for %s' % (cArg, argMarking))
```

Comment:
I think this will insert a big chunk of whitespace in the message? If so:
```suggestion
          gLogger.fatal(
              'Error when parsing command line arguments: '
              '"%s" does not match the allowed values for %s' % (cArg, argMarking)
          )
```